Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:latest
+
+ENV AWS_ACCESS_KEY_ID=accessKey1
+ENV AWS_SECRET_ACCESS_KEY=verySecretKey1
+
+# Copy certificates
+COPY tests/resources/certificates /usr/src/cvs-tsk-cert-gov-notify/certificates
+
+# Install dependencies
+RUN pip install --upgrade awscli \
+    && apt-get clean
+
+## Script from the web to wait for S3 to start up
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.2.1/wait /wait
+RUN chmod +x /wait
+
+## Run the wait script until SQS is up
+## Create buckets and add the signature
+## Start
+CMD /wait && \
+aws --endpoint-url=http://s3:7000 s3 mb s3://cvs-cert && \
```

Comment:
I think the devops team will request to add env variables in the bucket name (same as certificate generation microservice)

Reply:
I don't think it's necessary in the docker file as we are using it only locally.